### PR TITLE
[FIX] fix errors caused by & in protein accession

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

**Comment by timosachsenberg** [comment]
will add some additional escaping

**Comment by mwalzer** [comment]
#2608

File: src/tests/topp/IDFileConverter_16_output.idXML [hunk 1]
```diff
@@ -39,7 +39,7 @@
 			</PeptideHit>
 		</PeptideIdentification>
 		<PeptideIdentification score_type="InterProphet probability" higher_score_better="true" significance_threshold="0" MZ="566.77033723893" RT="0" >
-			<PeptideHit score="0.0505509" sequence=".(Glu->pyro-Glu)EAGIVDELAYA" charge="2" aa_before="K" aa_after="V" protein_refs="PH_4" >
+			<PeptideHit score="0.0505509" sequence=".(Glu-&gt;pyro-Glu)EAGIVDELAYA" charge="2" aa_before="K" aa_after="V" protein_refs="PH_4" >
```
Comment (hroest) [inline src/tests/topp/IDFileConverter_16_output.idXML]: that was probably not a good idea to have these characters as part of the  official openms names ...

Comment (timosachsenberg) [inline src/tests/topp/IDFileConverter_16_output.idXML]: they come from Unimod so we probably should blame them ;)

**Comment by timosachsenberg** [comment]
everyone fine with merge to master?

**Comment by jpfeuffer** [comment]
+1

**Pull request merged**